src/Sulu/Bundle/MediaBundle/Markup/MediaTag.php

```diff
@@ -144,9 +152,46 @@ function ($attributes) {
         $result = [];
         foreach ($medias as $media) {
             $media['url'] = $this->mediaManager->getUrl($media['id'], $media['name'], $media['version']);
+            $dispositionType = $this->findMediaDispositionType($media);
+
+            if ($dispositionType) {
+                $media['url'] .= '&' . $dispositionType . '=true';
+            }
+
             $result[$media['id']] = $media;
         }
 
         return $result;
     }
+
+    /**
+     * Find disposition type by media MIME type.
+     *
+     * @param array $media
+     *
+     * @return string|null
+     */
+    private function findMediaDispositionType(array $media)
```

Comment:
Would you really make this service aware of building an URL? I think leave it by `getDefault` and let the router decide where to put that information (some controllers might use the predefined value we would use in that function for something else).